editor/src/templates/editor.tsx

```diff
@@ -249,6 +249,7 @@ export class Editor {
       unpatchedDerived: derivedState,
       patchedDerived: derivedState,
       strategyState: strategyState,
+      postActionInteractionSession: null,
```

Comment:
the name feels too similar to the canvas interaction session but this one works very differently, I think we should name it something like `postActionMenuData` what do you think ?

Reply:
Maybe `postActionMenuSession` would be better? I think it's the `Interaction` part that is a bit confusing